src/components/network/gs-map-equipments.js

```diff
@@ -16,33 +16,14 @@ import {
     fetchSubstationsMapInfos,
 } from '../../services/study/network';
 import { MAX_NUMBER_OF_IMPACTED_SUBSTATIONS } from './constants';
+import { MapEquipments } from '@powsybl/diagram-viewer';
 
 const elementIdIndexer = (map, element) => {
     map.set(element.id, element);
     return map;
 };
 
-export default class MapEquipments {
-    substations = [];
-
-    substationsById = new Map();
-
-    lines = [];
-
-    linesById = new Map();
-
-    hvdcLines = [];
-
-    hvdcLinesById = new Map();
-
-    voltageLevels = [];
-
-    voltageLevelsById = new Map();
-
-    nominalVoltages = [];
-
-    intlRef = undefined;
-
+export default class GSMapEquipments extends MapEquipments {
     initEquipments(studyUuid, currentNodeUuid) {
```

Comment:
reloadImpactedSubstationsEquipments(), the constructor and initEquipments() can stay in GSMapEquipments yes. As they are specific and use the fetch methods. 
But updateEquipments(), completeSubstationsInfos(), updateSubstations(), updateLines() and completeLinesInfos() should be moved to MapEquipments in powsybl-diagram-viewer